### PR TITLE
Fixes for LLVM5

### PR DESCRIPTION
**Pull request by sklam**
Based on #306 

This updates llvmlite for the changes in LLVM 5:

* Fix C-API get attribute usage.
* Remove the deprecated get_pointer_to_function since it behavior is unexpected.

### CODE REVIEW

File: llvmlite/binding/value.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-
-from ctypes import POINTER, c_char_p, c_int
+import warnings
```
Comment (stuartarchibald) [inline llvmlite/binding/value.py]: Is this used?

Comment (sklam) [reply llvmlite/binding/value.py]: Good catch.  It's now fixed.

**Comment by stuartarchibald** [comment]
Thanks for the fixup... merging.

**Comment by jakirkham** [comment]
Any idea as to when this might land in a release?

**Comment by seibert** [comment]
We are hoping to take a version 0.36 release candidate next week, with the final release a week later.

**Comment by jakirkham** [comment]
Thanks @seibert. Could you please give me a ping when those land?

**Pull request merged**